app/src/main/res/layout-v28/activity_profile.xml

```diff
@@ -69,7 +69,7 @@
                         android:layout_toRightOf="@id/user_image"
                         android:gravity="start"
                         android:text="@string/user_name"
-                        android:textColor="#999999"
+                        android:textColor="#373737"
```

Comment:
Don't want hardcoded colors

Reply:
Updatedthe activity_profil in layout-v28 + backtesting done
Please delete the other activity_profil in layout in order to avoid any confusions